team-agreements/working-agreements/readme.md

```diff
@@ -1,15 +1,63 @@
 # Sections of a working agreement
 
+A working agreement is a document or a set of documents that describe how we work together as a team and what our expectations and principles are.
+
+The working agreement created by the team at the beginning of the project, and is stored in the repository so that it is readily available for everyone working on the project.
+
+The following are examples of sections and points that can be part of a working agreement but each team should compose their own, and adjust times etc. to fit their team needs.
+
+## General
+
+- We work as one team towards a common goal and clear scope
+- We make sure everyones voice is heard, listened to
+- We show all team members equal respect
+- We pair-program as often as possible
+- We make sure to spread our expertise and skills in the team, so no single person is relied on for one skill
+- All times below are listed in CET
+
+## Communication
+
+- We communicate all information relevant to the team through the Project Teams channel
+- We add all research results, design documents and other technical documentation to the project repository through PRs
+
+## Work life balance
+
+- Our office hour, when we can expect to collaborate via Microsoft Teams, phone or face-to-face are Monday to Friday 10AM-5PM
```

Comment:
same here about "Teams"

Reply:
Can we add a specific time here? I think that's probably something the team will agree on, no? Could also be expressed as a suggestion/example.

Reply:
See statement at the top of the doc: "The following are examples of sections and points that can be part of a working agreement but each team should compose their own, and adjust times, communication channels etc. to fit their team needs."